### ISSUE TITLE
[Epic] [macos][arm64] Tracking GtkSharp issues affecting Skia/Gtk

### ISSUE THREAD

**Issue by spouliot** [unoplatform/uno#8600]
### Current behavior

The latest release of GtkSharp has some issues on macOS/arm64 (M1). This issue tracks the issues: found / fixed / released affecting this configuration.

- [ ] https://github.com/GtkSharp/GtkSharp/pull/344
  This affects application startup (native libraries can't be found)
- [ ] https://github.com/GtkSharp/GtkSharp/pull/346
  This affects native file dialogs

### Expected behavior

An Uno app running on Skia/Gtk on an arm64-based Mac should behave identically as running on x86_64 Mac computer.

### How to reproduce it (as minimally and precisely as possible)

Run Uno Skia/Gtk apps on an arm64-based Mac.

### Workaround

Use the HEAD of the `develop` branch of `GtkSharp`.

### Environment

Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia

### Affected platforms

Skia (GTK on Linux/macOS/Windows)

### Anything else we need to know?

Some issues could also exists on other arm64 platforms (linux or windows).

**Comment by jeromelaban** [comment]
Thanks!

**Comment by MartinZikmund** [comment]
Can this be closed?

**Comment by spouliot** [comment]
@MartinZikmund no, I would left it open for reference. My fixes are merged but they were never released so the issues remain (unless you build gtk# yourself).

**Comment by MartinZikmund** [comment]
GTK is not supported
